.ci/appsettings.override.postgres.docker.json

```diff
@@ -166,5 +166,14 @@
                 }
             }
         ]
+    },
+    "WellKnownPreview": {
+        "AppleApps": ["1576693742.eu.enmeshed.app"],
```

Comment:
`1576693742` is incorrect, maybe you want to fix this, so that you don't apply this anywhere by accident. Same anywhere else.